CHANGELOG.md

```diff
@@ -4,6 +4,34 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/).
 
+## [0.6.0] - 2024-12-06
+
+### Added
+- **Prometheus recording rules** to improve query performance and reduce server load.
+- **Prometheus self-monitoring** for better observability of the monitoring system.
+- **Support for podman-compose** as a fallback when Docker Compose is not available.
+- **PostgreSQL log collection from the collector** for improved debugging capabilities.
+- **2-day time range option** in the time selector for extended analysis periods.
+- **Export snapshots via API** enabling external data access and integration.
+- **`--no-collector` mode** in run.sh for flexible deployment options.
+- **`--reprocess-full-snapshots` and `--reprocess-compact-snapshots` flags** for data reprocessing.
+
+### Changed
+- **Enhanced time selector UI** with improved usability and support for all time ranges.
+- **Optimized input data collection** and PromQL queries for better performance.
+- **Upgraded Prometheus** with improved query capabilities.
```

Comment:
This sounds like a Prometheus version upgrade. Is that right? Or is it switching to query fingerprints. I don't see that described anywhere.

Reply:
Yeah, this one is for the prometheus upgrade that's done.
Added another item for fingerprints.